x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/enrich/EnrichPolicy.java

```diff
@@ -268,16 +296,39 @@ public void writeTo(StreamOutput out) throws IOException {
         @Override
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
             builder.startObject();
+            builder.startObject(policy.type);
             {
                 builder.field(NAME.getPreferredName(), name);
```

Comment:
should name appear outside the policy type object? I assumed not, because otherwise we have the name and policy type at the same level.

Reply:
I feel like name should stay within the policy's definition. The object above this will only have a few fields that are considered valid: the existing policy types. On top of that, I think the previously decided direction is to keep other metadata about the policy (like the es version it was created under) contained in the definition part.